From 45d1268aa761bdd700f49419881ce7822efb6d6d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Motiejus=20Jak=C5=A1tys?= <motiejus.jakstys@spilgames.com>
Date: Wed, 3 Apr 2013 20:19:00 +0300
Subject: [PATCH 2/9] Respect DESTDIR parameter in "install" target

---
 Makefile |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index a7211d0..6e0b373 100644
--- a/Makefile
+++ b/Makefile
@@ -49,9 +49,9 @@ build-dir:
 
 
 install:
-	-install -d $(PIQI_PREFIX)/bin
-	install piqi-tools/piqi $(PIQI_PREFIX)/bin
-	install piqic/piqic $(PIQI_PREFIX)/bin
+	-install -d $(DESTDIR)$(PIQI_PREFIX)/bin
+	install piqi-tools/piqi $(DESTDIR)$(PIQI_PREFIX)/bin
+	install piqic/piqic $(DESTDIR)$(PIQI_PREFIX)/bin
 
 
 ocaml:
-- 
1.7.9.5

